MAUI/Cartesian-Charts/Axis/Types.md

```diff
@@ -334,6 +335,182 @@ chart.XAxes.Add(primaryAxis);
 
 ![DateTimeAxis range customization support in MAUI Chart](Axis_Images/maui_chart_datetime_axis_range.jpg)
 
+## Logarithmic Axis
+
+The [LogarithmicAxis]() uses a logarithmic scale, and it is very useful in visualising data when the given data range has a big difference. It can be used either on the x-axis or the chart's y-axis. The logarithmic axis cannot be plotted with zero or negative values. The following code example demonstrates how to set the logarithmic axis in the chart.
+
```

Comment:
The logarithmic axis cannot be plotted with zero or negative values. 

@RajaRamalingam3772 , remove above line.

Reply:
Changes updated.